zerver/decorator.py

```diff
@@ -349,12 +360,18 @@ def _wrapped_func_arguments(request: HttpRequest, api_key: str=REQ(),
             try:
                 return view_func(request, user_profile, *args, **kwargs)
             except Exception as err:
+                # NOTE: importing this at the top of the file leads to a
+                # cyclic import; correct fix is probably to move
+                # UnexpectedWebhookEventType to a different module.
```

Comment:
zerver/lib/exceptions.py is probably the right choice for that.

Reply:
It'd be potentially reasonable to move the definition and still have things import through `zerver.lib.webhooks.common`